src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -381,6 +381,11 @@ private bool CanUseTupleTypeName(INamedTypeSymbol tupleSymbol)
         {
             INamedTypeSymbol currentUnderlying = tupleSymbol.TupleUnderlyingType;
 
+            if (currentUnderlying.Arity == 1)
```

Comment:
Does this need to be <= 1?  Is there a similar issue with zero arity value tuples?